src/man/firejail.txt

```diff
@@ -319,6 +319,7 @@ Print content of file from sandbox container, see FILE TRANSFER section for more
 Chroot the sandbox into a root filesystem. Unlike the regular filesystem container,
 the system directories are mounted read-write. If the sandbox is started as a
 regular user, nonewprivs and a default capabilities filter are enabled.
+Disabled by default in firejail.config.
```

Comment:
Note that the defaults may change, such as with `private-lib` in #5190.

So to avoid having to also update a man page when changing a default, just note
what is the relevant configuration option:

```suggestion
Note: This command may be disabled by using "chroot no" in firejail.config.
```

Reply:
Indeed that's a better idea. As these are already disabled by default it makes more sense to detail how to enable them. Hence the rephrasing. Did the same for private-lib and tracelog. Thanks for the suggestion.

Reply:
@glitsj16 glitsj16 [on Sep 13](https://github.com/netblue30/firejail/pull/5366#discussion_r970075159):

> Thanks for the suggestion.

No problem.

> Indeed that's a better idea. As these are already disabled by default it
> makes more sense to detail how to enable them. Hence the rephrasing. Did the
> same for private-lib and tracelog.

The new version does look nice:

> ```
> Note: this command may be enabled in firejail.config with "chroot yes".
> ```

But "may be enabled" still implies that it is disabled by default, which
ideally would also be changed (for consistency) if/when the default changes in
firejail.config.

Note that this extra work affects not only us, but any downstream that happens
to change the defaults (such as with the Debian patch).  It seems trickier to
patch the man page (and to keep the patch working with man page updates)
compared to just running something like the following:

```sh
sed -i.bak 's/chroot no/chroot yes' etc/firejail.config
```

So how about just mentioning what the relevant option is in firejail.config?
Example:

```diff
-Note: this command may be enabled in firejail.config with "chroot yes".
+Note: Support for this command is controlled in firejail.config with the
+\fBchroot\fR option.
```